src/client/components/chart-data-buttons/chartbar-buttons/chart-data-button.component.js

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from 'react';
+import './chart-data-button.style.css';
+import PropTypes from 'prop-types';
+import Button from '../../button/button.component';
+
+const ChartbarMenu = ({ buttonLabel }) => {
+  const [selectedButton, setSelectedButton] = useState(buttonLabel.buttonWeek);
```

Comment:
Also according to the `story` this would be `undefined`  🙌

Reply:
yes